src/Configuration/Dotenv/Parse.hs

```diff
@@ -76,9 +76,11 @@ interpolatedValueVarInterpolation = VarInterpolation <$>
     symbol                = L.symbol sc
 
 interpolatedValueCommandInterpolation :: Parser VarFragment
-interpolatedValueCommandInterpolation =
-  CommandInterpolation
-    <$> between (symbol "$(") (symbol ")") (many alphaNumChar)
+interpolatedValueCommandInterpolation = do
+  ws <- between (symbol "$(") (symbol ")") ShellWords.parser
+  pure $ case ws of
+      (commandName:arguments) -> CommandInterpolation commandName arguments
+      _ -> CommandInterpolation "echo" [""]
```

Comment:
@pbrisbin Does this mean that if there are no arguments it will return the result of "echo ''"? What about using `VarLiteral ""` instead? It will cause the same effect, won't it?

Reply:
That's right. `VarLiteral ""` is indeed a simpler way to do that :man_facepalming: 

I meant to comment and ask you what you wanted `FOO=$()` to mean. If an empty value works for you, I'll make the change to `VarLiteral`.